src/components/Weather.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { currentWeather } from '../utils/current_weather';
+
+export default class Weather extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: null,
+      res: false,
+      temp: null,
+    };
+  }
+
+  componentDidMount() {
+    this.getWeather();
+  }
+
+  getWeather() {
+    currentWeather().then(res =>
+      this.setState({
+        data: res.data,
+        res: true,
+        temp: res.data.main.temp,
+      }));
+  }
+
+  render() {
+    if (this.state.res === false) return <div>Loading</div>;
```

Comment:
in here is fine but a common practice is to `{ res } = this.state` to access that